sdv/sampling/independent_sampler.py

```diff
@@ -98,16 +100,18 @@ def _finalize(self, sampled_data):
         final_data = {}
         for table_name, table_rows in sampled_data.items():
             synthesizer = self._table_synthesizers.get(table_name)
-            metadata = synthesizer.get_metadata()._convert_to_single_table()
+            if isinstance(synthesizer.metadata, Metadata):
+                metadata = synthesizer.metadata.tables[synthesizer._table_name]
+            else:
+                metadata = synthesizer.metadata
             dtypes = synthesizer._data_processor._dtypes
```

Comment:
Is this for Enterprise compatibility? Can we add an issue for addressing this change in Enterprise as well?

Reply:
Yes. What do you mean by addressing the change in Enterprise?

Reply:
Just a mirror issue to track making any necessary changes to enterprise for compatibility, basically the changes you made in the [draft PR](https://github.com/datacebo/SDV-Enterprise/pull/1130)